includes/definitions/adva.yaml

```diff
@@ -0,0 +1,12 @@
+os: adva
```

Comment:
I feel this is too generic. The os name should be based after the firmware/software the devices run, like IOS for Cisco switches.

If they don't have a name then at worst the os here should be something like adva-fsp.